CODEOWNERS

```diff
@@ -1 +1 @@
-* @C1KB-tensileai @P21I-Akira @AKDO-slalom @THOA-slalom @GPEC-adhoc @TUAS-forpeople @BKY3-forpeople @VPAI-bixal @TBWM-slalom @MIPI-forpeople @GHHZ-bixal
\ No newline at end of file
+* @C1KB-tensileai @FICU-tensileai@AKDO-slalom @THOA-slalom @GPEC-adhoc @TUAS-forpeople @BKY3-forpeople @VPAI-bixal @TBWM-slalom @MIPI-forpeople @GHHZ-bixal
```

Comment:
Needs a space between the user IDs.

Reply:
Goodness, I am sorry about that @TUAS-forpeople